vendor/assets/stylesheets/dvl/components/sub_navigation.scss

```diff
@@ -0,0 +1,52 @@
+// Secondary and tertiary navigation,
+// within page headers and subheaders.
```

Comment:
Are these actually used within `subheader`? 

Do we want to keep them as a separate component, which would mean supporting their use _outside_ of the page header, or should we integrate them into the page header component because it doesn't make sense to use them elsewhere?